File: backend/Modulos/models/paramGrabaDetallesOC.go
package models

// ParamGrabaDetallesOC is...
type ParamGrabaDetallesOC struct {
	NumeroDocOc      int     `json:"numerodococ"`
	OcDetTipoItem    string  `json:"ocdettipoitem"`
	OcDetMeInID      int     `json:"ocdetmeinid"`
	OcDetCantCalc    int     `json:"ocdetcantcalc"`
	OcDetCantReal    int     `json:"ocdetcantreal"`
	OcFechaAnulacion string  `json:"ocfechaanulacion"`
	OcDetValCosto    float64 `json:"ocdetvalcosto"`
	OcOrCoID         int     `json:"orcoid"`
	PiUsuario        string  `json:"usuario"`
	PiServidor       string  `json:"servidor"`
}

File: frontend/src/app/models/entity/ParamGrabaDetallesOC.ts
export class ParamGrabaDetallesOC{
    constructor (
        
        public numerodococ      ?: number,
        public ocdettipoitem    ?: string,
        public ocdetmeinid      ?: number,
        public ocdetcantcalc    ?: number,
        public ocdetcantreal    ?: number,
        public ocfechaanulacion ?: string,
        public ocdetvalcosto    ?: number,
        public orcoid           ?: number,
        public usuario          ?: string,
        public servidor         ?: string
    ) {}
}